src/services/expenses.service.js

```diff
@@ -0,0 +1,79 @@
+const { createId } = require('../utils/id.service');
+
+let expenses = [];
+
+const resetExpenses = () => {
+  expenses = [];
+};
+
+const getExpenses = () => {
+  return expenses;
+};
+
+const getExpense = (id) => {
+  return expenses.find((elem) => elem.id === id);
+};
+const addExpense = (elem) => {
+  const elemWithId = {
+    id: createId(expenses),
+    ...elem,
+  };
+
+  expenses.push(elemWithId);
+
+  return elemWithId;
+};
+const updateExpense = (newElem) => {
+  const { id } = newElem;
+
+  expenses = expenses.map((elem) => (elem.id === +id ? newElem : elem));
+
+  return newElem;
+};
+const deleteExpense = (id) => {
+  const index = expenses.findIndex((elem) => elem.id === id);
```

Comment:
Similar to the previous comment, ensure that the `id` parameter is treated as a number. Consider using `+id` to ensure consistency.